src/index.html

```diff
@@ -13,24 +13,25 @@
     <main>
         <header>
             <h1>Leaderboard</h1>
+            <h4 id="timeDate"></h4>
         </header>
+        
         <div class="board">
         <div class="score">
-            <div class="score-board-header"><h2>Recent Scores</h2><input type="button" id="RefreshBtn" value="Refresh"></div>
+            <div class="score-board-header"><h2>Recent Scores</h2><h3 id="counter" title="Scores"></h3><input type="button" id="RefreshBtn" value="Refresh"></div>
             <div id="scores"></div>
         </div>
         <div class="score-form">
             <div class="score-form-header"><h2>Add Your Score</h2></div>
             <form action="post" id="score-form">
-               <label for="names"> <input type="text" id="names" placeholder="Your Name">
-               <label for="scoressTXT"> <input type="text" id="scoresTXT" placeholder="Your Scores"><br>
-                <input type="button" class="" id="btnAdd" value="Sumit">
+               <label for="names"></label> <input type="text" id="names" placeholder="Your Name">
+               <label for="scoressTXT"></label> <input type="text" id="scoresTXT" placeholder="Your Scores"><br>
+                <input type="button" class="btnAdd" id="btnAdd" value="Submit">
             </form>
         </div>
         </div>
     </main>
 
 <script src="main.js"></script>
-    <!-- <script src="index.js" type="module"></script> -->
```

Comment:
- [ ] Kindly remove unused commented codes. As well as this one:

<img width="599" alt="Screen Shot 2022-10-13 at 07 33 21" src="https://user-images.githubusercontent.com/69011963/195485573-18591bb5-aa8f-4e40-9f48-47996a147a98.png">